### DISCUSSION TITLE
Foundry docker image open-source?

### DISCUSSION THREAD

**Discussion by alandixon** [foundry-rs/foundry#8874] [Help] [answered]
Hi,
Is the Foundry docker image open-source?

More specifically, I am writing a blockchain challenge for a CTF (Capture the Flag) competition and I think the docker image would be a good place to start as it has much of the functionality I need.

If that is viable, what kind of licence is it created under and what attribution text would I need to use, if any?

Sorry if the answer is obvious, I have searched the Github pages and I'm not sure.

Thanks!

**Comment by zerosnacks** [comment, marked as answer]
Yes the Docker image is open source

Foundry is released under a dual https://github.com/foundry-rs/foundry/blob/master/LICENSE-APACHE / https://github.com/foundry-rs/foundry/blob/master/LICENSE-MIT license

For attribution something like this could work

```
Foundry
[Foundry GitHub repository](https://github.com/foundry-rs/foundry)
Licensed under the [Apache License 2.0](https://github.com/foundry-rs/foundry/blob/master/LICENSE-APACHE)
```

**Reply by alandixon** [reply]
Great. Thanks for your help.
